app/views/admin/users/index.html.erb

```diff
@@ -0,0 +1,56 @@
+<div class='container my-5'>
+  <div class='row'>
+    <div class='col-12'>
+
+      <%= link_to [:new, :admin, :user], class: 'btn btn-primary float-right mb-3' do %>
+        <%= icon('plus') %>
+        <%= I18n.t('users.add') %>
+      <% end %>
+
+      <div class='table-responsive'>
+        <table class='table table-bordered'>
+          <thead>
+            <tr>
+              <th>
+                <%= I18n.t('user.name') %>
+              </th>
+              <th>
+                <%= I18n.t('user.email') %>
+              </th>
+              <th style="width: 70px;">
+              </th>
+            </tr>
+          </thead>
+          <tbody>
+            <% @users.each do |user| %>
+              <% cache user do %>
+                <tr>
+                  <td>
+                    <%= user.name %>
+                    <% if user.admin? %>
+                      <br>
+                      Admin
```

Comment:
Admin could be in a separate column. Useful in case wanting to sort later.

Reply:
yep I considered that. Just added it as quick way to sport admins in the table for now. I'll separate it if  / when sorting is added.